lib/measures/tbd/measure.rb

```diff
@@ -325,10 +325,11 @@ def run(mdl, runner, args)
         end
       end
 
-      unless kva
-        argh[:gen_kiva  ] = false if argh[:gen_kiva  ]
-        argh[:kiva_force] = false if argh[:kiva_force]
-      end
+      return false unless kva
+        # argh[:gen_kiva  ] = false if argh[:gen_kiva  ]
+        # argh[:kiva_force] = false if argh[:kiva_force]
+        # return false
+      # end
```

Comment:
EDIT (original comment: _"If massless (or otherwise invalid) materials/constructions are targeted for KIVA input generation, raise informative ERROR messages and ignore user-required KIVA processes altogether."_). After testing a number of scenarios using the measure (e.g. App, CLI), revising TBD feedback/outcome policy when faced with massless materials + KIVA (from _informative_ to _failure_).